sdk/php/template/composer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "daggermodule/example",
+  "description": "An example module you can use as a starting point for building your own dagger modules",
+  "minimum-stability": "stable",
+  "license": "proprietary",
+  "authors": [
+  ],
+  "repositories": [
+    {
+      "type": "path",
+      "url": "./sdk"
+    }
+  ],
+  "require": {
+    "php": "^8.1",
```

Comment:
bump this to PHP 8.3 because of introducing `json_validate()`

Reply:
We're not using json_validate as the implementation moved on from there. 8.2 should be fine; we can bump it in future when 8.2 is not under active support.